testing/citest/tests/kube_smoke_test.py

```diff
@@ -112,7 +112,12 @@ def __init__(self, bindings, agent=None):
     # because it scopes the context of our activities.
     # pylint: disable=invalid-name
     self.TEST_APP = bindings['TEST_APP']
-
+    # This postfix is required when we create stateful/daemonset , if we add the existing framework application name
+    # such as XXXXXXXXXX-tst-v001 its giving error when we are tring to delete it, hence we are removing '-' in between
```

Comment:
I still dont understand where "tst" comes from. Is that the application name?
If so, you should have something more unique to this test so it can coexist with other tests running concurrently. You dont want to have different test scenarios using the same application name and stomping on each others assumptions about external state or they are going to break in mysterious hard to replicate ways.

Reply:
You already using word 'tst' in your framework see below 
  defaults = {
      'TEST_STACK': 'tst',
      'TEST_APP': 'kubsmok' + KubeSmokeTestScenario.DEFAULT_TEST_ID
  }
Its a TEST_STACK , not a application name so it will not be coexists any where else as i explained in the comment  we are removing '-' in between so it will be unique for statefulset/daemonset , will not be used any where else.